.github/workflows/images.yml

```diff
@@ -77,6 +77,8 @@ jobs:
           # Tag with 'latest'
           # if the branch is 'main'
           publish_latest: ${{ steps.tags.outputs.tag_latest }}
+          # If we are publishing latest, also tag it with calver
+          publish_calver: ${{ steps.tags_outputs.tag_latest }}
```

Comment:
For any push to a branch changing a docker image, build and push the hash. If it's main, then also push latest and calendar versions.